pkg/networkpolicy/networkpolicy.go

```diff
@@ -131,11 +131,11 @@ func (builder *NetworkPolicyBuilder) WithPolicyType(policyType netv1.PolicyType)
 	}
 
 	if builder.Definition.Spec.PolicyTypes == nil {
-		builder.Definition.Spec.PolicyTypes = make([]netv1.PolicyType, 1)
+		builder.Definition.Spec.PolicyTypes = []netv1.PolicyType{policyType}
+	} else {
```

Comment:
Please remove else and add return
```
if builder.Definition.Spec.PolicyTypes == nil {
    builder.Definition.Spec.PolicyTypes = []netv1.PolicyType{policyType}
    retrun builder
}

builder.Definition.Spec.PolicyTypes = append(builder.Definition.Spec.PolicyTypes, policyType)

return builder
```

Reply:
I have no idea why you prefer it like this